samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -239,11 +239,19 @@ public async Task<string> AcquireExternalInformationAsync(SKContext context)
             var plan = Plan.FromJson(planJson, newPlanContext);
 
             // Invoke plan
-            SKContext planContext = await plan.InvokeAsync(plannerContext);
+            try
```

Comment:
This try/catch resets the callstack of the exception. Is this intentional or can we remove the try/catch?

Reply:
Changed it to just a throw which preserves the callstack